compiler/test/formatter_outputs/function_params.gr

```diff
@@ -3,3 +3,13 @@ let single_arg = x => x
 let unit_arg = () => 3
 
 let two_args = (x, y) => 4
+
+export let fd_write: (WasmI32, WasmI32, WasmI32, WasmI32) -> WasmI32 =
+  (
+    fd,
+    iovs,
+    iovs_len,
+    nwritten,
+  ) => {
+  0n
+}
```

Comment:
Formatter doesn't put a newline at eof?

Reply:
I remember when reading the tests that all line endings are removed because rely doesn't like them. So this isn't (can't be?) tested.

However, I believe this is already a setting in things like vscode, so it would be weird if we'd do it. It could cause constant conflicts if our decision was different than the user's editor settings.

Reply:
@phated is correct there is a newline at the end, but there's a bug in a Reason/Jest interaction, I can't where I raised the issue, that needs fixing.

But hexdump of the formatter output:

0000080 7b 0a 20 20 30 6e 0a 7d 0a

so it ends in a newline